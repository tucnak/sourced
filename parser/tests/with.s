with ALT {
	alias alright {
		say "hello"
	}

	bind H {
		alright
	}
}
