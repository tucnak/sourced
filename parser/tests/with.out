alias +with_ALT "alias alright "say "hello"";bind H "alright""
alias -with_ALT "unalias alright;unbind H"
bind ALT +with_ALT
